### DISCUSSION TITLE
Fix Hitcircle sizing on ios

### DISCUSSION THREAD

**Discussion by h3llo-wor1d** [ppy/osu#17627] [Ideas]
Something I'm seeing as a persistent issue is that it is using the 1080p screen that is small to put circles that I can barely see, letalone hit. Having a 2x magnification on iphone would be cool. I'm already using beatmap diff adjustment, but I just think scaling in general would be good

**Comment by bdach** [comment]
https://github.com/ppy/osu/issues/4888
https://github.com/ppy/osu/issues/6490

**Reply by h3llo-wor1d** [reply]
Yeah but that was like 3 years ago, isn't lazer going live like really, really soon?

**Reply by bdach** [reply]
mobile version is not a priority

**Comment by peppy** [comment]
You can apply the "Difficulty Adjust" mod for now to make circles bigger.

**Reply by h3llo-wor1d** [reply]
> You can apply the "Difficulty Adjust" mod for now to make circles bigger.

I just wanted to ask if it was in production, since std is the only mode that works and iirc you posted that lazer would be going live soon, was just curious about where the mobile development was at in that case :p
